src/main/java/duke/command/Command.java

```diff
@@ -2,9 +2,13 @@
 
 import duke.task.TaskList;
 
```

Comment:
Try not to leave any random blank lines. Not sure if it's just the GitHub showing the lines but otherwise okay.